multigeiger/webconf.cpp

```diff
@@ -105,9 +104,38 @@ void handleRoot(void) {  // Handle web requests to "/" path.
   server.send(200, "text/html;charset=UTF-8", index);
 }
 
+static char lastWiFiSSID[IOTWEBCONF_WORD_LEN] = "";
+
+void loadConfigVariables(void) {
+  // check if WiFi SSID has changed. If so, restart cpu. Otherwise, the program will not use the new SSID
+  if ((strcmp(lastWiFiSSID, "") != 0) && (strcmp(lastWiFiSSID, iotWebConf.getWifiSsidParameter()->valueBuffer) != 0)) {
+    log(INFO, "Doing restart...");
+    ESP.restart();
+  }
+  strcpy(lastWiFiSSID, iotWebConf.getWifiSsidParameter()->valueBuffer);
+
+  speakerTick = strcmp(speakerTick_c, "1") == 0;
```

Comment:
ok for now, i'ld add something a bit better later.

Reply:
OK